### PR TITLE
Add EthereumAddress to PortableRegistry

### PR DESCRIPTION
**Pull request by joelamouche**
As per recommanded in https://github.com/polkadot-js/api/issues/4110#issuecomment-951808119

### CODE REVIEW

File: packages/types/src/generic/PortableRegistry/PortableRegistry.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ const PATHS_PRIMITIVE = splitNamespace([
   'pallet_democracy::vote::Vote',
   'pallet_identity::types::Data',
   // these are well-known types with additional encoding
+  'core_primitives::EthereumAddress',
```
Comment (jacogr) [inline packages/types/src/generic/PortableRegistry/PortableRegistry.ts:36]:
```suggestion
  'account::AccountId20',
```

this seems aligned with the Moonbeam PR, please double-check.

Comment (jacogr) [reply packages/types/src/generic/PortableRegistry/PortableRegistry.ts:36]: As per https://github.com/PureStake/moonbeam/pull/926#discussion_r739500374

**Comment by crystalin** [comment]
@jacogr is there more to do for this PR before merging it ?

**Comment by jacogr** [comment]
Nothing more - if we have the path correct, we can merge.

File: packages/types/src/generic/PortableRegistry/PortableRegistry.ts (hunk 1 above)
Comment (jacogr) [reply packages/types/src/generic/PortableRegistry/PortableRegistry.ts:36]: Going to merge as-is, with this change to make the Monday release. If it changes, we need an update.

**Comment by crystalin** [comment]
Sorry @jacogr ,
@joelamouche is off this week and created the repo in his private account :( so I couldn't modify it.

**Comment by jacogr** [comment]
No issues. Breaks are good ;)

**Comment by jacogr** [comment]
Closing, replaced by https://github.com/polkadot-js/api/pull/4175 (see comment in PR)

**Pull request closed without merging**